### PR TITLE
moves AFK/stealthmin handling to the server, and refactors the discordchannel config

### PR DESCRIPTION
**Pull request by nichlas0010**
also adds a snowflake check for whether or not it's an admin channel, but I honestly do not want to edit the config more than necessary

### CODE REVIEW

**Comment by nichlas0010** [comment]
needs https://github.com/yogstation13/Yogstation-TG/pull/1288

File: models/Discord/Commands/DiscordCommandAdminWho.js [hunk 1]
```diff
@@ -10,20 +10,16 @@ class DiscordCommandAdminWho extends DiscordCommand {
   onRun(message, permissions, args) {
     var config = this.subsystem.manager.getSubsystem("Config").config;
     var byondConnector = this.subsystem.manager.getSubsystem("Byond Connector").byondConnector;
+		var byondmessage = "?adminwho";
 
-    byondConnector.request("?adminwho", (results) => {
+		if(message.channel.id == "134722036353204224" || message.channel.id == "378318242906636288" || message.channel.id == "134722015503319040") { //admin, admemes and council respectively
```
Comment (AshCorr) [inline models/Discord/Commands/DiscordCommandAdminWho.js]: Should use config file for these channels.

File: models/Discord/Commands/DiscordCommandAdminWho.js [hunk 2]
```diff
@@ -10,20 +10,16 @@ class DiscordCommandAdminWho extends DiscordCommand {
   onRun(message, permissions, args) {
     var config = this.subsystem.manager.getSubsystem("Config").config;
     var byondConnector = this.subsystem.manager.getSubsystem("Byond Connector").byondConnector;
+		var byondmessage = "?adminwho";
```
Comment (AshCorr) [inline models/Discord/Commands/DiscordCommandAdminWho.js]: indentation is wrong

File: models/Discord/Commands/DiscordCommandInfo.js [hunk 3]
```diff
@@ -13,8 +13,12 @@ class DiscordCommandInfo extends DiscordCommand {
 		var config = this.subsystem.manager.getSubsystem("Config").config;
 		var byondSS = this.subsystem.manager.getSubsystem("Byond Connector");
 		var discord = this.subsystem.manager.getSubsystem("Discord").discord;
+		var byondmessage = "?adminwho";
+		
+		if(message.channel.id == "134722036353204224" || message.channel.id == "378318242906636288" || message.channel.id == "134722015503319040") { //admin, admemes and council respectively
```
Comment (AshCorr) [inline models/Discord/Commands/DiscordCommandInfo.js]: config file.

File: models/Discord/Commands/DiscordCommandAdminWho.js [hunk 4]
```diff
@@ -10,20 +10,16 @@ class DiscordCommandAdminWho extends DiscordCommand {
...
 
-    byondConnector.request("?adminwho", (results) => {
+		if(message.channel.id == "134722036353204224" || message.channel.id == "378318242906636288" || message.channel.id == "134722015503319040") { //admin, admemes and council respectively
+			byondmessage += "&adminchannel=1"
+ 
+		byondSS.byondConnector.request(byondmessage, (resultsadmin) => {
```
Comment (AshCorr) [inline models/Discord/Commands/DiscordCommandAdminWho.js]: ban indentation

**Comment by nichlas0010** [comment]
@AshCorr there you go, now it's using a config :^)

**Comment by AshCorr** [comment]
I'm merging this because nich was complaining that I wouldn't merge it within 24 hours. So now I have to merge to say I told you so.

Probably shouldnt update the bot.

**Pull request merged**